src/bundle/Resources/public/js/scripts/helpers/content.type.helper.js

```diff
@@ -37,7 +37,26 @@
         return iconUrl;
     };
 
+    /**
+     * Returns contentType name from contentType identifier
+     *
+     * @function getContentTypeName
+     * @param {String} contentTypeIdentifier
+     * @returns {String|null} contentType name
+     */
+    const getContentTypeName = (contentTypeIdentifier) => {
+        let contentType;
+        for (var sectionContentTypes of Object.values(eZ.adminUiConfig.contentTypes)) {
```

Comment:
Instead of iterating always on the contentTypes you can use the `contentTypesDataMap` here.